src/ducks/connections/account.js

```diff
@@ -0,0 +1,18 @@
+import {
+  CREATE_CONNECTION,
+  UPDATE_CONNECTION_RUNNING_STATUS
+} from './'
+
+const reducer = (state = {}, action) => {
+  switch (action.type) {
+    case CREATE_CONNECTION:
+    case UPDATE_CONNECTION_RUNNING_STATUS:
+      const account = { ...state }
+      account.running = action.running
+      return account
```

Comment:
I cannot understand the instruction `const account = { ...state }`.
I write it down in [jsbin](http://jsbin.com/yepeceheco/edit?js,console) and it seems useless.

The classic way to write this down is:

```js
return { ...state, running: action.running }
```

Reply:
Well spotted @enguerran, I knew this one and it's a pure copy/paste typo.